notebooks-auth/test_thredds.ipynb

```diff
@@ -0,0 +1,554 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Test THREDDS using Magpie Authentication/Authorization against Twitcher proxy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Configuration parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function\n",
+    "import os\n",
+    "import uuid\n",
+    "import requests\n",
+    "print(\"Setup configuration parameters...\")\n",
+    "\n",
+    "PAVICS_HOST = os.getenv(\"PAVICS_HOST\", \"\").rstrip(\"/\")\n",
```

Comment:
Like the rest of the notebooks (ex: https://github.com/Ouranosinc/pavics-sdi/blob/master/docs/source/notebooks/pavics_thredds.ipynb), you can use env var but please provide `pavics.ouranos.ca` as default value.

Reply:
done